handlers/access_log.go

```diff
@@ -65,6 +69,11 @@ func (a *accessLog) ServeHTTP(rw http.ResponseWriter, r *http.Request, next http
 	alr.BodyBytesSent = proxyWriter.Size()
 	alr.StatusCode = proxyWriter.Status()
 	alr.RouterError = proxyWriter.Header().Get(router_http.CfRouterError)
+	alr.DNSTime = reqInfo.DNSTime
```

Comment:
as commented above, wouldn't it be easier to maintain if we just added one `AttemptsDetails` struct to the reqInfo?